inference-engine/src/inference_engine/ie_core.cpp

```diff
@@ -35,6 +35,8 @@ struct Parsed {
 
 template <typename T = Parameter>
 Parsed<T> parseDeviceNameIntoConfig(const std::string& deviceName, const std::map<std::string, T>& config = {}) {
+    OV_ITT_SCOPED_TASK(itt::domains::IE_LT, "parseDeviceNameIntoConfig");
```

Comment:
it's quite lightweight operation. I think we should not trace it

Reply:
removed